jj/mock/__init__.py

```diff
@@ -20,7 +20,6 @@
 from ._remote_mock import RemoteMock
 from ._remote_response import RemoteResponseType
 from ._stacked import stacked
-from ._system_log_filter import SystemLogFilter
```

Comment:
This change breaks backward compatibility. For instance, the code snippets provided in the README will no longer work as expected.